ranger-server-bundle/src/test/java/io/appform/ranger/server/bundle/RangerServerBundleTest.java

```diff
@@ -49,15 +45,15 @@
 
 public class RangerServerBundleTest {
 
-    private final JerseyEnvironment jerseyEnvironment = mock(JerseyEnvironment.class);
-    private final MetricRegistry metricRegistry = mock(MetricRegistry.class);
-    private final LifecycleEnvironment lifecycleEnvironment = new LifecycleEnvironment(metricRegistry);
-    private final Environment environment = mock(Environment.class);
-    private final Bootstrap<?> bootstrap = mock(Bootstrap.class);
-    private final Configuration configuration = mock(Configuration.class);
+    private static final JerseyEnvironment jerseyEnvironment = mock(JerseyEnvironment.class);
```

Comment:
rename to UPPER_SNAKE format for static members

Reply:
Done.